cap-ci/cap-pre-release-upgrades.yaml

```diff
@@ -38,6 +40,9 @@ schedulers:
 catapult:
   uri: https://github.com/SUSE/catapult
   branch: master
+mits:
+  uri: https://github.com/SUSE/minibroker-integration-tests.git
+  branch: master
```

Comment:
We should consume the GitHub release instead of the master branch. We can use https://github.com/SUSE/minibroker-integration-tests/releases/latest.

Reply:
this is not a resource for mits but just resource to get ./deploy/minibroker.sh and ./deploy/mits.sh
for mits this is the resource https://github.com/SUSE/cap/pull/49/files#diff-ee018afdb1f8e22c23a5298ff734fdc1R89-R93
which points to release

Reply:
The release contains the scripts in the source code tarball. We should use that instead. If the Helm chart for MITS changes, the script is tightly coupled with it in terms of knowing how to deploy. Using one branch to deploy another git commit (you can think in terms of another branch or another git tag as well) can lead to inconsistencies and brings undesirable non-deterministic behaviour.

Reply:
Done https://github.com/SUSE/cap/pull/49/commits/d450c442f159461cd7bfc9262196797151610a0b